tests/logs.go

```diff
@@ -100,7 +100,11 @@ func Logs(o *option.Option) {
 					gomega.Expect(session.Out.Contents()).Should(gomega.BeEmpty())
 					command.Run(o, "exec", testContainerName, "sh", "-c", fmt.Sprintf("echo %s >> /proc/1/fd/1", newLog))
 					output := strings.TrimSpace(string(session.Out.Contents()))
-					gomega.Expect(output).Should(gomega.Equal(newLog))
+					// allow propagation time
+					gomega.Eventually(output).
```

Comment:
Do we need to use `session.Out.Contents()` here instead? `output` might be empty if there was no data in the buffer when `output` was assigned.

Reply:
Fixed in latest revision